lnwallet/btcwallet/signer.go

```diff
@@ -26,17 +26,7 @@ func (b *BtcWallet) FetchInputInfo(prevOut *wire.OutPoint) (*wire.TxOut, error)
 		err    error
 		output *wire.TxOut
 	)
-
-	// First check to see if the output is already within the utxo cache.
-	// If so we can return directly saving a disk access.
-	b.cacheMtx.RLock()
-	if output, ok := b.utxoCache[*prevOut]; ok {
-		b.cacheMtx.RUnlock()
-		return output, nil
-	}
-	b.cacheMtx.RUnlock()
-
-	// Otherwise, we manually look up the output within the tx store.
```

Comment:
Could use a commit message explaining why this needs to be removed and doesn't have a negative effect?

Reply:
Done.